src/extras/helpers/BoxHelper.js

```diff
@@ -22,16 +22,10 @@ THREE.BoxHelper.prototype.constructor = THREE.BoxHelper;
 
 THREE.BoxHelper.prototype.update = function ( object ) {
 
-	var geometry = object.geometry;
+	var box = new THREE.Box3().setFromObject(object);
```

Comment:
Thanks, @gero3. I'd suggest a closure here, instead. This can be called in the render loop. Also, mrdoobapproves formatting.